src/Router.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import {BrowserRouter, Routes, Route} from "react-router-dom";
+import Introduce from "./Introduce/Introduce";
+import Product from "./pages/Product/Product"
+import ProductList from "./pages/ProductList/ProductList";
+
+
+const Router = () => {
+    return(
+        <BrowserRouter>
+        <Routes>
+            <Route path ="/" element = {<Introduce/>}/>
+            <Route path ="/Product" element = {<Product/>}/>
+            <Route path ="/ProductList" element = {<Introduce/>}/>
```

Comment:
```suggestion
            <Route path ="/" element = {<Introduce/>}/>
            <Route path ="/product" element = {<Product/>}/>
            <Route path ="/productList" element = {<Introduce/>}/>
```
경로도 위의 리뷰와 마찬가지로 소문자로 시작하도록 구현해주세요!

Reply:
소문자 변경 및 컴포넌트 연결 완료하였습니다.